examples/graphql-interfaces/graphql_interfaces.md

```diff
@@ -1,8 +1,6 @@
 # GraphQL service - Interfaces
 
-A GraphQL schema can have interfaces. An interface specifies a set of fields that multiple object types can include. In Ballerina, interfaces are defined using `distinct` `service` objects, and the fields of the interfaces are defined as resource method definitions. Objects that are implementing the interfaces must implement the `resource` methods defined in the service objects. The Ballerina type inclusion includes the interface type to an object type. Use an interface to return a type that consists of a set of possible types with common fields.
-
-This example shows how to define an interface `Profile` and then implement the `Teacher` and `Student` classes using that interface.
+The Ballerina GraphQL module allows defining GraphQL interface types. An interface specifies a set of fields that multiple object types can include. In Ballerina, interfaces are defined using `distinct` `service` objects, and the fields of the interfaces are defined as resource method definitions. Objects that are implementing the interfaces must implement the `resource` methods defined in the `service` objects. The Ballerina type inclusion includes the interface type to an object type. Use an interface to return a type that consists of a set of possible types with common fields.
```

Comment:
```suggestion
The Ballerina GraphQL module allows defining GraphQL interface types. An interface specifies a set of fields that multiple object types can include. In Ballerina, interfaces are defined using `distinct` `service` objects and the fields of the interfaces are defined as resource method definitions. Objects that are implementing the interfaces must implement the `resource` methods defined in the `service` objects. The Ballerina type inclusion includes the interface type to an object type. Use an interface to return a type that consists of a set of possible types with common fields.
```
This paragraph is too lengthy. Better to break it into two.

Reply:
@praneesha Is there a limitation on how many words should be in BBE description paragraph ?